.github/workflows/build.yml

```diff
@@ -11,10 +11,50 @@ jobs:
   build:
     name: Build binaries
     runs-on: ubuntu-latest
+    outputs:
+      version_major: ${{ steps.build_info.outputs.version_major }}
+      version_minor: ${{ steps.build_info.outputs.version_minor }}
+      version_build: ${{ steps.build_info.outputs.version_build }}
```

Comment:
Aren't the three versions required?
If yes, we can remove them.

Reply:
It is not required currently. Will need this for further community build checking.